sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -19,6 +19,29 @@
     secondaryLabel="Previous section"
     class="mx-0"
   >
+    <template #primary-buttons v-if="isLastPage">
+      <span>
+        <v-btn
+          color="primary"
+          v-if="!notDraft && !isFirstPage && !processing"
+          variant="outlined"
+          :loading="savingDraft"
+          @click="handleSaveDraft()"
```

Comment:
We can directly emit on click, instead of using a method. Same for line 38.

![image](https://github.com/user-attachments/assets/e5e603c3-c2ba-4288-99ed-8f85825d7036)